common/src/main/java/tachyon/exception/ExceptionMessage.java

```diff
@@ -100,7 +100,17 @@
   HDFS_FILE_NOT_FOUND("File {0} with id {1} is not found"),
 
   // file system master
+  FILE_MUST_HAVE_VALID_PARENT("{0} does not have a valid parent"),
   FILEID_MUST_BE_FILE("File id {0} must be a file"),
+  RENAME_CANNOT_BE_ONTO_MOUNT_POINT("{0} is a mount point and cannot be renamed onto"),
```

Comment:
Without the `onto` it seems like you are trying to rename the mount point and not something else to the mount point? Though I'm not a big fan and would be happy to change it to something better.